utils/timezone.jsx

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import UserStore from 'stores/user_store.jsx';
+import * as UserActions from 'actions/user_actions.jsx';
+
+const dateTimeFormat = new Intl.DateTimeFormat();
+
+export function getSupportedTimezones() {
+    if (global.mm_config && global.mm_config.SupportedTimezones) {
```

Comment:
We are removing all global.mm_config usage in the application, you should use something like `const config = getConfig(store.getState())`

Reply:
Although `getConfig(store.getState())` is a correct workaround given that `global.mm_config` is now gone, note that it won't connect a component to future changes in the `SupportedTimezones`. We're aiming to eventually remove the required refresh when a configuration change occurs.

If possible, pass in the `config.supportedTimezones` to this method and invoke it from the connector. If that's not straightforward, the approach @jespino describes will suffice for now :)

Reply:
This should be turned into a selector in the redux library

Reply:
This has been moved to a selector 👍